src/OpenSage.Game/Logic/Object/Update/AutoFindHealingUpdate.cs

```diff
@@ -4,7 +4,28 @@ namespace OpenSage.Logic.Object
 {
     public sealed class AutoFindHealingUpdate : UpdateModule
     {
-        private uint _unknown;
+        private readonly AutoFindHealingUpdateModuleData _moduleData;
+
+        public AutoFindHealingUpdate(AutoFindHealingUpdateModuleData moduleData)
+        {
+            _moduleData = moduleData;
+        }
+
+        private LogicFrameSpan _framesTillNextScan;
```

Comment:
nit: `Until` or `Til` (I'd prefer the former personally) (to much of a grammar queen to not say anything lol)